dataland-frontend/tests/e2e/specs/quality-assurance/CheckQaProcedure.ts

```diff
@@ -94,6 +94,14 @@ function testSubmittedDatasetIsInReviewListAndAcceptIt(storedCompany: StoredComp
   cy.get('[data-test="qa-review-section"] .p-datatable-tbody tr').last().click();
 
   cy.get('[data-test="qaRejectButton"').should("exist");
+  cy.get('div[data-test="hideEmptyDataToggle"]').should("exist");
+  cy.get("span[class=p-inputswitch-slider]").should("exist");
+  cy.get('div[data-test="dataPointToggleButton"]').should("have.class", "p-inputswitch p-component");
+  cy.get('div[data-test="dataPointToggleButton"]').should(
+    "not.have.class",
+    "p-inputswitch p-component p-inputswitch-checked",
```

Comment:
` cy.get('div[data-test="dataPointToggleButton"]').should(
    "not.have.class",
    "p-inputswitch p-component p-inputswitch-checked");` is dangerous.
This assertion would also fail if the classes would be `p-component p-inputswitch p-inputswitch-checked`. You should only check for a single class .should("not.have.class", "p-inputswitch-checked").

Reply:
where?

Reply:
Ok, but this was not changed here.